.env.local

```diff
@@ -10,6 +10,9 @@ DEFAULT_SESSION_EXPIRATION=3600
 
 ACCOUNT_WEB_URL=http://localhost:3001
 
+FILE_TRANSFER_API_HOST='https://49d6uvlsrc.execute-api.eu-west-1.amazonaws.com'
+FILE_TRANSFER_API_KEY='wtG2I1FCuP1GIv3kYNJM53e3bP3YgJuIa2IF0ZDL'
```

Comment:
Please delete immediately, it should never be added to public repository. Wipe it of the history as well.